### ISSUE TITLE
Statsmodels import error

### ISSUE THREAD

**Issue by arokem** [nipy/dipy#1844]
On current PRs (e.g., https://github.com/nipy/dipy/pull/1843), we are seeing the following error: 

```
    def __getattr__(self, attr_name):
        """ Raise informative error accessing attributes """
>       raise TripWireError(self._msg)
E       dipy.utils.tripwire.TripWireError: We need package statsmodels.formula.api for these functions, but ``import statsmodels.formula.api`` raised an ImportError
```

For example in: https://travis-ci.org/nipy/dipy/jobs/536509460#L6102

I believe this is coming from the linear mixed model work.

@BramshQamar : do you understand this error? Is this something about the version of statsmodels that we are using?

**Comment by skoudoro** [comment]
I spend some time on this problem and I have currently no clue:
- There is no recent release for statsmodel (the last one was on May 15, 2018)
- our function `optional_package` did not change

I need to create the same environment ...

**Comment by skoudoro** [comment]
fix by @ariel on #1855, it was due to the last Scipy release. closing
